spec/payment/dispute_spec.rb

```diff
@@ -28,25 +19,12 @@
   end
 
   context '#evidence' do
-    let(:evidence) { dispute.evidence }
-
     describe 'attributes' do
-      it { expect(evidence).to respond_to(:product_description) }
-      it { expect(evidence).to respond_to(:customer_name) }
-      it { expect(evidence).to respond_to(:customer_email_address) }
-      it { expect(evidence).to respond_to(:billing_address) }
-      it { expect(evidence).to respond_to(:receipt) }
-      it { expect(evidence).to respond_to(:customer_signature) }
-      it { expect(evidence).to respond_to(:customer_communication) }
-      it { expect(evidence).to respond_to(:uncategorized_file) }
-      it { expect(evidence).to respond_to(:uncategorized_text) }
-      it { expect(evidence).to respond_to(:service_date) }
-      it { expect(evidence).to respond_to(:service_documentation) }
-      it { expect(evidence).to respond_to(:shipping_address) }
-      it { expect(evidence).to respond_to(:shipping_carrier) }
-      it { expect(evidence).to respond_to(:shipping_date) }
-      it { expect(evidence).to respond_to(:shipping_documentation) }
-      it { expect(evidence).to respond_to(:shipping_tracking_number) }
+      subject { dispute.evidence }
+
+      ShorePayment::Evidence::SUPPORTED_ATTRIBUTES.each do |attr|
+        it { is_expected.to respond_to(attr) }
+      end
```

Comment:
With this change, I don't see why we should have this test. If an attribute is removed, the test will still pass. If we change the list completely the test will still pass. In my understanding that's the entire reason for all the 15 tests.